### PR TITLE
Excecute acctions with enter key

### LINKED ISSUE #413: Excecute acctions with enter key
To improve usability we could allow the user to execute the main action of that section by pressing the enter key

### PR DESCRIPTION
**Pull request by AngeloCG97**
## Excecute acctions with enter key

closes #413

### What does this PR do?
- Add in Login
- Add in credential component
- Add function in add image and add telephone in edit profile sponsor
- Add function in add image and add telephone in edit profile lifebankc

### How should this be tested?

- Run make run
- Go to localhost:3000
- Go to credential component, login, edit profile sponsor, edit profile lifebank

### CODE REVIEW

File: webapp/src/components/CredentialsRecovery/CredentialsRecovery.js [hunk 1]
```diff
@@ -183,6 +183,17 @@ const CredentialsRecovery = ({ overrideBoxClass, overrideLabelClass }) => {
     }
   }, [responseChangePassword])
 
+  function executeCredentialsRecovery(e) {
+    if (e.key === 'Enter' && ((user.newPassword && user.currentPassword && validEmailFormat) && !loadingChangePassword)) {
+      e.preventDefault();
```
Comment (JustinCast) [inline webapp/src/components/CredentialsRecovery/CredentialsRecovery.js:188]: Unnecessary semicolon here

File: webapp/src/components/CredentialsRecovery/CredentialsRecovery.js [hunk 2]
```diff
@@ -183,6 +183,17 @@ const CredentialsRecovery = ({ overrideBoxClass, overrideLabelClass }) => {
...
+      handleSubmitChangePassword()
+    }
+    else if (e.key === 'Enter' && (validEmailFormat && !loading)) {
+        e.preventDefault();
```
Comment (JustinCast) [inline webapp/src/components/CredentialsRecovery/CredentialsRecovery.js:192]: Unnecessary semicolon here

File: webapp/src/components/LoginModal/Login.js [hunk 3]
```diff
@@ -256,6 +255,13 @@ const LoginModal = ({ isNavBar, isSideBar }) => {
 
   }, [currentUser])
 
+  function executeLogin(e) {
+    if (e.key === 'Enter' && (user.account && user.secret && !loading)) {
+        e.preventDefault();
```
Comment (JustinCast) [inline webapp/src/components/LoginModal/Login.js:260]: Unecessary semicolon here

File: webapp/src/routes/EditProfile/EditProfileBank.js [hunk 4]
```diff
@@ -235,6 +235,33 @@ const EditProfileBank = ({ profile, isCompleting, onSubmit, setField, loading, u
 
   }, [isUnique, firstRun])
 
+  function executeAddImage(e) {
+    if (e.key === 'Enter' && (!disablePhotoUrlInput)) {
+        e.preventDefault();
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileBank.js:240]: Unnecessary semicolon here

File: webapp/src/routes/EditProfile/EditProfileSponsor.js [hunk 5]
```diff
@@ -198,6 +198,33 @@ const EditProfileSponsor = ({ profile, isCompleting, onSubmit, loading }) => {
       : ''
   }
 
+  function executeAddImage(e) {
+    if (e.key === 'Enter' && (!disablePhotoUrlInput)) {
+        e.preventDefault();
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileSponsor.js:203]: Unecessary semicolon here

File: webapp/src/routes/EditProfile/EditProfileSponsor.js [hunk 6]
```diff
@@ -198,6 +198,33 @@ const EditProfileSponsor = ({ profile, isCompleting, onSubmit, loading }) => {
...
+          photos: [...user.photos, photoUrlValueRef.current.value]
+        })
+        photoUrlValueRef.current.value = ''
+        setDisablePhotoUrlInput(true)
+    }
+  }
+
+  function executeAddTelephone(e) {
+    if (e.key === 'Enter' && (!disablePhoneInput)) {
+        e.preventDefault();
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileSponsor.js:215]: Unecessary semicolon here

File: webapp/src/routes/OffersManagement/GenericOfferFormComponent.js [hunk 7]
```diff
@@ -288,6 +288,18 @@ const GenericOfferFormComponent = ({
     }
   }, [createOfferResult])
 
+  function executeAddImage(e) {
+    if (e.key === 'Enter' && (!disableUrlInput)) {
+        e.preventDefault();
```
Comment (JustinCast) [inline webapp/src/routes/OffersManagement/GenericOfferFormComponent.js:293]: Unecessary semicolon here

**Review by JustinCast — CHANGES_REQUESTED** [review]

**Review by JeanVegaD — APPROVED** [review]

**Review by JustinCast — APPROVED** [review]

**Pull request merged**